gpu/utils/include/pcl/gpu/utils/timers_cuda.hpp

```diff
@@ -60,8 +60,8 @@ namespace pcl
                 cudaEventDestroy(stop_);
             }
 
-            void start() { cudaEventRecord(start_, 0); }
-            Timer& stop()  { cudaEventRecord(stop_, 0); cudaEventSynchronize(stop_); return *this; }
+            void start() { cudaEventRecord(start_, nullptr); }
```

Comment:
Second argument is `cudaStream_t`, a stream identifier. Technically it's a pointer, so I understand why `clang-tidy` made this change. However, in CUDA SDK stream identifiers don't have pointer semantics. It's just an identifier, but how exactly it is implemented we (users) should not care. And the meaning of `0` is not a "null pointer", but the default stream.

Same for `cudaStreamSynchronize`.